library/ui/src/main/java/com/google/android/exoplayer2/ui/DebugTextViewHelper.java

```diff
@@ -18,25 +18,46 @@
 import android.annotation.SuppressLint;
```

Comment:
I can leave this file out of the review.  Changes are to expose the upstream as well as the downstream level and the current playback position in US.